### ISSUE TITLE
Character Map scenario does not display wide runes correctly

### ISSUE THREAD

**Issue by tig** [gui-cs/Terminal.Gui#1693]
**Describe the bug**
Related to #1685, the Character Map scenario does not display all wide runes correctly. See gif:

**To Reproduce**
(This is with #1685)
![image](https://i.imgur.com/FEZskz9.gif)

**Comment by tig** [comment]
Oops. Realized my gif showing the issue got posted as a png. Fixed.

**Comment by BDisp** [comment]
But this issue is waiting for the NStack fix, right?

**Comment by tig** [comment]
> But this issue is waiting for the NStack fix, right?

If you say so. I just wanted to ensure it was clearly tracked since it is an a good example of remaining wide rune issues.

Do you have a private fix for nstack that I can try to prove to myself that the Nstack fix does fix this?

**Comment by BDisp** [comment]
> If you say so. I just wanted to ensure it was clearly tracked since it is an a good example of remaining wide rune issues.
> 
> Do you have a private fix for nstack that I can try to prove to myself that the Nstack fix does fix this?

Yes. In this comment https://github.com/migueldeicaza/gui.cs/pull/1685#issuecomment-1115496655

**Comment by tig** [comment]
Closing as a Dupe of #4233
